dataset_builder/humaneval_to_ocaml.py

```diff
@@ -74,7 +74,7 @@ def coerce_to_option(expr: str) -> str:
 
 class Translator:
 
-    stop = ["\n\n", "\n(*", "\ntype"]
+    stop = ["\n\n", "\n(*", "\ntype", "\nlet"]
```

Comment:
Suggested other stop tokens: `\nmodule`, `;;` Opinions @JohnGouwar ?

Reply:
_Technically_ OCaml no longer requires `;;` but it's in a lot of historical code so I could see the argument. I feel like `\n\n` covers most keywords that we would tend to care about, since it would tend to precede `module`, `type`, `sig`, `let`, etc. It's also more future proof to any other keywords that get added in OCamls 5+